### PR TITLE
5e Shaped ITA translation

### PR DESCRIPTION
**Pull request by 00xeno00**
Translation for the 5e Shaped chara sheet.

### CODE REVIEW

**Comment by Alzam01** [comment]
You should probably push that into @mlenser 's repo (https://github.com/mlenser/roll20-character-sheets) so he can check/change things.

File: 5eShaped/precompiled/translations/it.json [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+    "STILLNESS_OF_MIND": "Quiete mentale",
+    "STILLNESS_OF_MIND_TEXT": "Puoi usare la tua azione per terminare un effetto su di te che ti stia causando l'effetto affascinato o spaventato.",
+    "STROKE_OF_LUCK": "Colpo di fortuna",
+    "STROKE_OF_LUCK_TEXT": "Hai una prodigiosa predisposizione per riuscire nel momento del bisogno. Se il tuo attacco manca un bersaglio entro la portata o il raggio di azione, puoi trasformarlo in un attacco riuscito. In alternativa, se fallisci una prova di abilità, puoi trattare il risultato del d20 come se fosse 20.",
+    "STUNNING_STRIKE": "Colpo stordente",
+    "STUNNING_STRIKE_TEXT": "Puoi interferire con il flusso del ki nel corpo di un avversario. Quando colpisci un'altra creatura con un attacco con arma da mischia, puoi spendere 1 punto ki per tentare un colpo stordente. Il bersaglio deve riuscire un tiro salvezza di Costituzione o restare stordito fino al termine del tuo prossimo turno.",
+    "STUNNING_STRIKE_SAVING_THROW_FAILURE": o restare stordito fino al termine del tuo prossimo turno",
```
Comment (Alzam01) [inline 5eShaped/precompiled/translations/it.json]: Looks like a " is missing in front of the translated text

File: 5eShaped/precompiled/translations/it.json [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+  "SUCCESSES": "Successi",
+  "SWIM": "Nuotare",
+  "TARGET": "Bersaglio",
+  "TOTAL": "Totale",
+  "TRACKER": "Contatore",
+  "TRAITS": "Tratti",
+  "TREMORSENSE": "Senso tellurico",
+  "TRUESIGHT": "Visione del vero",
+  "TYPE": "Tipo",
+  "TURN_OFF_FILTERING": Se non riesci a vedere incantesimi aggiunti di recente disattiva \"filters\"",
```
Comment (Alzam01) [inline 5eShaped/precompiled/translations/it.json]: A " is missing in front of the translated text

Comment (00xeno00) [inline 5eShaped/precompiled/translations/it.json]: Fixed it, thanks!

**Comment by mlenser** [comment]
This has to be done on my branch:

https://github.com/mlenser/roll20-character-sheets/tree/master/5eShaped

**Comment by 00xeno00** [comment]
I tried to fork from yours but it doesn't let me.

**Comment by mlenser** [comment]
You can only have 1 fork, correct. You'll have to delete your fork on the main branch and then fork mine.

You can copy your translations to a different notepad or something to prevent losing them.

**Pull request closed without merging**